From 82fb858df896388d9e0c6b4db261c4896ce57b0c Mon Sep 17 00:00:00 2001
From: chromium-autoroll
 <chromium-autoroll@skia-public.iam.gserviceaccount.com>
Date: Mon, 25 Feb 2019 04:51:36 +0000
Subject: [PATCH] Roll src/third_party/skia 52d8a5976a23..e1636a4b56aa (1
 commits)

https://skia.googlesource.com/skia.git/+log/52d8a5976a23..e1636a4b56aa


git log 52d8a5976a23..e1636a4b56aa --date=short --no-merges --format='%ad %ae %s'
2019-02-25 skia-autoroll@skia-public.iam.gserviceaccount.com Roll third_party/externals/swiftshader 7c33e88eebb5..b47c816ad7e6 (3 commits)


Created with:
  gclient setdep -r src/third_party/skia@e1636a4b56aa

The AutoRoll server is located here: https://autoroll.skia.org/r/skia-autoroll

Documentation for the AutoRoller is here:
https://skia.googlesource.com/buildbot/+/master/autoroll/README.md

If the roll is causing failures, please contact the current sheriff, who should
be CC'd on the roll, and stop the roller if necessary.

CQ_INCLUDE_TRYBOTS=luci.chromium.try:android_optional_gpu_tests_rel;luci.chromium.try:linux-blink-rel;luci.chromium.try:linux-chromeos-compile-dbg;luci.chromium.try:linux_optional_gpu_tests_rel;luci.chromium.try:mac_optional_gpu_tests_rel;luci.chromium.try:win_optional_gpu_tests_rel
TBR=scroggo@chromium.org

Change-Id: Ibfd8e27303ff84d3bef7dea1da0cd3d7ee60c1eb
Reviewed-on: https://chromium-review.googlesource.com/c/1485496
Reviewed-by: chromium-autoroll <chromium-autoroll@skia-public.iam.gserviceaccount.com>
Commit-Queue: chromium-autoroll <chromium-autoroll@skia-public.iam.gserviceaccount.com>
Cr-Commit-Position: refs/heads/master@{#635031}
---
 DEPS | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/DEPS b/DEPS
index 7fd6a9b0c2290..72a967cb09cda 100644
--- a/DEPS
+++ b/DEPS
@@ -127,7 +127,7 @@ vars = {
   # Three lines of non-changing comments so that
   # the commit queue can handle CLs rolling Skia
   # and whatever else without interference from each other.
-  'skia_revision': '52d8a5976a23ad4b14ff3e824dd7ad4460d2cdfd',
+  'skia_revision': 'e1636a4b56aacd0e9f26bf7d0dd2a2f14afe2bd3',
   # Three lines of non-changing comments so that
   # the commit queue can handle CLs rolling V8
   # and whatever else without interference from each other.
